internal/controller/vrg_volrep.go

```diff
@@ -2916,12 +2916,20 @@ func (v *VRGInstance) HandleSecondaryConflictsAndCleanup() bool {
 	if !v.IsDRActionInProgress() {
 		v.log.Info("Skip resource cleanup; reconcile as secondary")
 
+		v.logAndSetConditions(VRGConditionTypeAutoCleanup,
```

Comment:
`logAndSetConditions` was specifically to aggregate conditions reported by the different sub-systems, vr and volsync. Using it here is not required. We just need to update the required condition in VRG as per latest observation. (also other callers below)